SymPEP-0001.md

```diff
@@ -0,0 +1,270 @@
+# SymPEP 1 --- SymPEP Purpose and Process
+
+**Author** Aaron Meurer, Jason K. Moore  
+**Status** Draft  
+**Type** Process  
+**Created** 2021-01-27  
+**Resolution**  
+
+## Abstract
+
+SymPEPs are a formal process whereby important changes are proposed and
```

Comment:
```suggestion
SymPEPs are a formal process whereby important controversial changes are proposed and
```

I just read this "Matplotlib Enhancement Proposals (MEP), inspired by cpython's [PEP's](https://www.python.org/dev/peps/) but less formal, are design documents for large or controversial changes to Matplotilb." and liked the word "controversial". This may help people defer to a normal PR or mailing list discussion before coming to make a SYMPEP.

Reply:
I think this is the most significant point that's missing here, and it really deserves more text than just a single word. It isn't clear just what sorts of changes require a SymPEP and which changes can just be an issue or PR. I really think the default should be to not have a SymPEP, at least for codebase changes (process/governance changes are different). This isn't CPython. We don't need a proposal just to add or change anything. 

Actually I'm not sure "controversial" is the right word to use here. A very big change may be uncontroversial, at least in principle, but still require a SymPEP because it requires a lot of coordination, or because we need to get everyone on the same page with the details.

Of course, controversial things should be discussed as well, but I'm a little unclear what that means. If something is controversial, shouldn't it just be rejected, because that's by definition a lack of community consensus.